code/game/jobs/job/security.dm

```diff
@@ -154,8 +154,10 @@
 	access = list(access_security, access_sec_doors, access_brig, access_court, access_maint_tunnels, access_morgue, access_weapons)
 	minimal_access = list(access_security, access_sec_doors, access_brig, access_court, access_maint_tunnels, access_weapons)
 	minimal_player_age = 14
+	alt_titles = list("Law Enforcement Officer")
```

Comment:
They're not the law or law enforcement; they're security, this really isn't applicable on a private research station.